pyaerocom/colocation/colocation_utils.py

```diff
@@ -681,10 +681,12 @@ def colocate_gridded_ungridded(
     """
     if filter_name is None:
         filter_name = const.DEFAULT_REG_FILTER
-    try:
-        data.check_dimcoords_tseries()
-    except DimensionOrderError:
-        data.reorder_dimensions_tseries()
+    if data.proj_info is None:
+        # reordering only implemented if no projection
```

Comment:
This try/except block appears several times but I forget what the reordering of the dimensions does. Could you please extend your comment to make a note of what it's doing and why this only happens for non-projected data.

Reply:
Sorry, I just implemented the code for projected data and left the code for non-projected data untouched. I can only guess from function names that the old code tries to reorder to "time, level, lon, lat" if this is not the natural order of data - not sure if this is a iris-requirement or where this old code came from.

I could add some synthetic test-data which might raise some problems when not ordered, but all real data we have doesn't have this problem. So I would rather only implement re-ordering of dimensions if this becomes an issue.

Reply:
Sounds good, don't worry about it then.